src/inc/msquicp.h

```diff
@@ -65,6 +65,26 @@ typedef struct QUIC_PRIVATE_TRANSPORT_PARAMETER {
     const uint8_t* Buffer;
 } QUIC_PRIVATE_TRANSPORT_PARAMETER;
 
+#define QUIC_SSLKEYLOG_MAX_SECRET_LEN 64
+typedef struct QUIC_SSLKEYLOG {
+    uint8_t ClientRandom[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ClientEarlyTrafficSecret[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ClientHandshakeTrafficSecret[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ServerHandshakeTrafficSecret[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ClientTrafficSecret0[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ServerTrafficSecret0[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t EarlyExporterSecret[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ExporterSecret[QUIC_SSLKEYLOG_MAX_SECRET_LEN];
+    uint8_t ClientRandomLength;
+    uint8_t ClientEarlyTrafficSecretLength;
```

Comment:
Are all the secrets same length (if present)? If so, might be best to have one length and bitfield for which are set?